worlds/bomb_rush_cyberfunk/__init__.py

```diff
@@ -0,0 +1,212 @@
+    
+
+    def get_filler_item_name(self) -> str:
+        item = self.random.choice(item_table)
+
+        while self.get_item_classification(item["type"]) == ItemClassification.progression:
+            item = self.random.choice(item_table)
+
+        return item["name"]
+
+
+    def generate_early(self):
+        grafM = group_table["graffitim"]
+        grafL = group_table["graffitil"]
+        grafXL = group_table["graffitixl"]
+
+        self.selectedM = self.random.choice(grafM)
+        self.selectedL = self.random.choice(grafL)
+        self.selectedXL = self.random.choice(grafXL)
+
+        if self.options.starting_movestyle == StartStyle.option_skateboard:
+            self.item_classification[BRCType.Skateboard] = ItemClassification.filler
+        else:
+            self.item_classification[BRCType.Skateboard] = ItemClassification.progression
+
+        if self.options.starting_movestyle == StartStyle.option_inline_skates:
+            self.item_classification[BRCType.InlineSkates] = ItemClassification.filler
+        else:
+            self.item_classification[BRCType.InlineSkates] = ItemClassification.progression
+        
+        if self.options.starting_movestyle == StartStyle.option_bmx:
+            self.item_classification[BRCType.BMX] = ItemClassification.filler
+        else:
+            self.item_classification[BRCType.BMX] = ItemClassification.progression
+
+
+    def create_items(self):
+        rep_locations: int = 86
+        if self.options.skip_polo_photos:
+            rep_locations -= 18
+
+        self.options.total_rep.round_to_nearest_step()
+        rep_counts = self.options.total_rep.get_rep_item_counts(self.multiworld.random, rep_locations)
+        #print(sum([8*rep_counts[0], 16*rep_counts[1], 24*rep_counts[2], 32*rep_counts[3], 48*rep_counts[4]]), \
+        #    rep_counts)
+
+        pool = []
+
+        for item in item_table:
+            if "REP" in item["name"]:
+                count: int = 0
+
+                if item["name"] == "8 REP":
+                    count = rep_counts[0]
+                elif item["name"] == "16 REP":
+                    count = rep_counts[1]
+                elif item["name"] == "24 REP":
+                    count = rep_counts[2]
+                elif item["name"] == "32 REP":
+                    count = rep_counts[3]
+                elif item["name"] == "48 REP":
+                    count = rep_counts[4]
+
+                if count > 0:
+                    for _ in range(count):
+                        pool.append(self.create_item(item["name"]))
+            else:
+                pool.append(self.create_item(item["name"]))
+
+        self.multiworld.itempool += pool
+
+
+    def get_stage(self, stage: BRCStage, player: int) -> Region:
+        return self.multiworld.get_region(region_names[stage], player)
+
+
+    def create_regions(self):
+        world = self.multiworld
```

Comment:
```suggestion
        multiworld = self.multiworld
```
Please do not do world = self.multiworld

Reply:
Noted